libs/domain/feature-loaders/src/loaders.ts

```diff
@@ -35,5 +36,6 @@ export const createLoaders = (apiKeys: ApiKeys) =>
         createPOAPTokenLoader(),
         createPOAPAccountLoader(),
         createPOAPTransferLoader(),
+        createSnapshotProposalLoader("banklessvault.eth"), // TODO: remove magic string
```

Comment:
Should I go ahead and make this a env variable in this PR or make a seperate ticket for that?